### PR TITLE
removed unused var

### PR DESCRIPTION
**Pull request by voisardf**
Fixes the pdf archiving for the standard call using the geoportal interface.

### CODE REVIEW

File: crdppf/views/printproxy.py [hunk 1]
```diff
@@ -109,17 +109,18 @@ def report_create(self):
...
+                archive_path = self.config['pdf_archive_path']
+            except:
+                archive_path = None
+
+            if archive_path is not None:
+                import os
+                with open(os.path.join(archive_path, outputFilename+'.pdf'), 'wb') as f:
+                    f.write(print_result.body)
+            else:
+                print 'Optional archive_path not set.'
```
Comment (kalbermattenm) [inline crdppf/views/printproxy.py]: It would be nicer to use logging.... instead of printing....

File: crdppf/views/printproxy.py [hunk 2]
```diff
@@ -148,4 +149,17 @@ def report_get(self):
             ),
         )
 
+        try:
+            archive_path = self.config['pdf_archive_path']
+        except:
+            archive_path = None
+
+        if archive_path is not None:
+            import os
```
Comment (kalbermattenm) [inline crdppf/views/printproxy.py]: This should be on top of the file.

File: crdppf/views/printproxy.py [hunk 3]
```diff
@@ -148,4 +149,17 @@ def report_get(self):
...
+        except:
+            archive_path = None
+
+        if archive_path is not None:
+            import os
+            filename = pdf.content_disposition.split('=')[1]
+            with open(os.path.join(archive_path, filename), 'wb') as f:
+                f.write(pdf.body)
+        else:
+            print 'Optional archive_path not set.'
```
Comment (kalbermattenm) [inline crdppf/views/printproxy.py]: Again logging would be nicer

**Comment by voisardf** [comment]
fixed as asked. thanks :)

**Pull request merged**